improv/nexus.py

```diff
@@ -59,14 +62,13 @@ def createNexus(
         control_port = int(
             self.in_socket.getsockopt_string(SocketOption.LAST_ENDPOINT).split(":")[-1]
         )
-
         self._startStore(
             store_size
         )  # default size should be system-dependent; this is 40 GB
         self.out_socket.send_string("Store started")
-
         # connect to store and subscribe to notifications
-        self.store = Store()
+        logger.info("Create new store object")
+        self.store = Store(store_loc=self.store_loc)
```

Comment:
Again, if we move the uuid creation inside `Store`, then we could just assign `self.store_loc = store.store_loc` or use `store.store_loc` everywhere.